README.md

```diff
@@ -2,8 +2,6 @@
 
 This repository contains the source code for the blockchain nodes used in the DSCP project. The structure and code is heavily based on [Substrate Node Template](https://github.com/substrate-developer-hub/substrate-node-template). To use this repository, it's important to understand the [key concepts](https://substrate.dev/docs/en/) of Substrate, such as `FRAME`, `runtime`, `extrinsics` and `transaction weight`.
 
-For more on governance visit [Governance](governance.md)
```

Comment:
Change this so it instead links to the dscp-documentationb repository page for governance.

Reply:
done